site/_posts/2017-08-11-back-to-basics.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Last Week in ManageIQ: Back to Basics"
+author: jntullo
+date: 2017-08-15
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! It's [Jillian](https://github.com/jntullo) again, here with the latest happenings the last *two* weeks in ManageIQ. 
```

Comment:
> here with the latest happenings the last *two* weeks in ManageIQ

I feel like something may be missing here.
"here with the latest happenings (of / for / in) the last *two* weeks in ManageIQ"